core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2059,6 +2074,11 @@ public String toString() {
     public static final String PROXY_WEB_PORT = "alluxio.proxy.web.port";
 
     //
+    // Log server related properties
+    //
+    public static final String LOG_SERVER_BASE_LOGS_DIR = "alluxio.logserver.baselogsdir";
```

Comment:
I think our convention is to dot separate words.

Reply:
In this case for log server, "log" and "server" can mean totally different things when they are separated. Therefore I'm hesitant about separating them.

Reply:
however we separate it, we should separate the string with dots the same way we separate the variable name with underscores